### ISSUE TITLE
Staging Review finding: Submit question button not responding to keyboard when using at least one screen reader

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#104825]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Ask VA Integration
**Product:** Ask VA
**Feature:** Ask Va

## Findings details
**VA.gov Experience Standard - issue:** [User can complete necessary flows.](https://design.va.gov/about/experience-standards/#user-can-complete-necessary-flows)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** Yes
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

I'm noticing some odd behavior when testing with at least one screen reader.

When the screen reader is active, the "Submit question" button on [/ask-va/review-then-submit](https://staging.va.gov/contact-us/ask-va/review-then-submit) didn't respond to expected keystrokes and would only work if clicking with a mouse. When this happened, I noticed that my screen reader would also announce the button as unavailable, which is also what it does for the disabled state.

<details><summary>Link, screenshot or steps to recreate</summary>

Steps to reproduce:
1. Log in with a test user.
2. Activate a screen reader (recommend NVDA on Windows or VoiceOver on Mac for easiest testing).
3. Complete the form.
4. On the review page, press [tab] until keyboard focus is on the Submit question button. Note that the button is announced as unavailable.
5. Press [enter]. Note that no action occurs.

</details>

## Recommended action

Here's my best guess for what's happening: The submit button is coded as...
```
<va-button text="Submit question" disabled="false" class="hydrated"></va-button>
```

But looking in Storybook, I'm seeing the `va-button` component has just `disabled` when disabled and no prop at all when not disabled, rather than having the prop assinged `true` or `false` values. Screen readers sometimes do funny things with certain ARIA attributes in similar circumstances. It may be worth trying just `<va-button text="..." class="...">` with no prop, then adding only `disabled` when it's supposed to be disabled.

It's also possible that something strange is happening with my device. If you're not able to reproduce this behavior you're probably safe to close.

## References

- **Accessibility Defect Severity:** [1: Critical. Must be fixed before launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-1)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [x] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by arielma-tw** [comment]
Issue has been addressed with https://github.com/department-of-veterans-affairs/ask-va/issues/1657
